src/subapps/admin/components/Settings/ViewsSubView.tsx

```diff
@@ -236,7 +235,51 @@ const ViewsSubView = () => {
     }
   );
 
-  const columns: ColumnsType<TViewType> = [
+  const fetchIndexingErrorsOnDemand = async ({
+    nexus,
+    apiEndpoint,
+    orgLabel,
+    projectLabel,
+    viewId,
+  }: {
+    nexus: NexusClient;
+    apiEndpoint: string;
+    orgLabel: string;
+    projectLabel: string;
+    viewId: string;
+  }) => {
+    try {
+      // Fetch indexing errors for the specified view
+      const indexingErrors = await fetchIndexingErrors({
+        nexus,
+        apiEndpoint,
+        orgLabel,
+        projectLabel,
+        viewId,
+      });
+
+      return indexingErrors;
+    } catch (error) {
+      console.error('Error fetching indexing errors on demand', error);
```

Comment:
Just curious - Why is this console needed here? If it's for error tracking purposes, I think the Sentry exception below would suffice so we can remove this.